dev-tools/ecs-migration.yml

```diff
@@ -1211,6 +1211,43 @@
 
 ## Modules
 
+### System
+
+- from: system.socket.direction
```

Comment:
@simianhacker Could you see that and of the fields change in this file could have an effect on Infra UI?